README.rst

```diff
@@ -38,8 +38,8 @@ When you use the BluePyEModel software or method for your research, we ask you t
       month        = aug,
       year         = 2023,
       publisher    = {Zenodo},
-      doi          = {},
-      url          = {}
+      doi          = {8283491},
+      url          = {https://doi.org/10.5281/zenodo.8283491}
```

Comment:
Could you use this? https://doi.org/10.5281/zenodo.8283490
So that it still points to the latest version, even when bpem is updated? (link was under 'cite all version' of zenodo website)